vita3k/config/src/config.cpp

```diff
@@ -249,6 +251,8 @@ void merge_configs(Config &lhs, const Config &rhs, const std::string &new_pref_p
         lhs.hardware_flip = rhs.hardware_flip;
     if (lhs.performance_overlay != rhs.performance_overlay && (!init || rhs.performance_overlay))
         lhs.performance_overlay = rhs.performance_overlay;
+    if (lhs.backend != rhs.backend && (!init || !rhs.backend.empty()))
```

Comment:
change here also by
```suggestion
      if (lhs.backend_render != rhs.backend_render && (!init || rhs.backend_render != static_cast<std::string>("OpenGL")))
```

Reply:
I don't like OpenGL being the empty option.

Reply:
set OpenGL include config and set like that here